agasc/healpix.py

```diff
@@ -108,7 +108,7 @@ def get_stars_from_healpix_h5(
     stars : astropy.table.Table
         Table of stars within the search cone, with columns from the AGASC data table.
     """
-    from agasc import read_h5_table, sphere_dist
+    from agasc import read_h5_table, sphere_dist  # noqa: PLC0415
```

Comment:
We've since updated skare3 ruff-base to exclude PLC0415 anyway, so you can leave the noqa or update base.